engine/src/main/java/org/hibernate/search/engine/common/impl/MappedIndexSearchTargetImpl.java

```diff
@@ -83,6 +84,26 @@ public String toString() {
 		);
 	}
 
+	@Override
+	public <P, T, Q> SearchQueryResultContext<Q> queryAsProjections(SessionContextImplementor sessionContext,
```

Comment:
Shouldn't it be `queryAsProjection()` without an s?

Reply:
Looks like this one got forgotten or was it intentional?

Reply:
I renamed the method with the same name in PojoSearchTargetDelegate and forgot there was also this one. And of course that's the only one I forgot to check with a global search afterwards...

Well spotted. Fixed.